build/source/driver/multi_driver.f90

```diff
@@ -225,18 +331,22 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 ! *****************************************************************************
 ! (5b) read Noah vegetation and soil tables
 ! *****************************************************************************
+
 ! define monthly fraction of green vegetation
 !                           J        F        M        A        M        J        J        A        S        O        N        D
 greenVegFrac_monthly = (/0.01_dp, 0.02_dp, 0.03_dp, 0.07_dp, 0.50_dp, 0.90_dp, 0.95_dp, 0.96_dp, 0.65_dp, 0.24_dp, 0.11_dp, 0.02_dp/)
```

Comment:
Clearly not part of this PR - but just FMI - why is this hardcoded? May be good to open an issue if this needs to be replaced.